instaclone/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import "./App.css";
+import PostsPage from "./components/PostContainer/PostsPage.js";
+import Authenticate from "./components/Authentication/Authenticate.js";
+import Login from "./components/Login/Login";
+
+class App extends Component {
```

Comment:
No need for a class component here if you're not utilizing the constructor or state